src/modules/corp-type-module/corp-type-module.ts

```diff
@@ -116,7 +116,8 @@ const CorpInfoArray: Array<CorpInfoIF> = [
     colinInd: true,
     corpClass: CorpClass.BC,
     shortDesc: 'BC CCC',
-    fullDesc: 'BC Community Contribution Company'
+    fullDesc: 'BC Community Contribution Company',
+    numberedDesc: 'Numbered Community Contribution Company'
```

Comment:
Because this was missing, the breadcrumbs and entity info header didn't display properly for a numbered CCC.